test/E2ETest/Infrastructure/ConditionalTheoryAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+using Xunit.Sdk;
+
+namespace E2ETest
+{
+    using System;
```

Comment:
Fix the usings to be out of the namespace..

Reply:
Fixed